connector/connect/server/src/main/scala/org/apache/spark/sql/connect/planner/SparkConnectPlanner.scala

```diff
@@ -1154,6 +1155,91 @@ class SparkConnectPlanner(val session: SparkSession) {
         }
         Some(Lead(children.head, children(1), children(2), ignoreNulls))
 
+      case "bloom_filter_agg" if fun.getArgumentsCount == 5 =>
```

Comment:
TBH I think we need to get rid of all this manual function resolution, and try to fold it back into the FunctionRegistry.

Reply:
`bloom_filter_agg ` is an internal function that does not allow users to directly use it, `fold it back into the FunctionRegistry.` may require some additional work